.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+| Q                   | A
+| ------------------- | -------
+| Target Release?     | <!-- which release milestone is this for? -->
+| Bug fix?            | yes/no <!-- fixes an issue -->
+| New feature?        | yes/no <!-- adds a new feature/behavior change -->
+| Database migration? | yes/no <!-- lets us know to look for migrations -->
+| New configuration?  | yes/no <!-- lets us know if we need new config items -->
+| Jira Tickets        | HPRO-... <!-- Tag which ticket(s) this PR relates to -->
+
```

Comment:
How about adding some suggested headings, like:

```md
### Summary

### Instructions for testing  <!-- if applicable -->
(Comments below could go here)

### Additional details <!-- if applicable -->

### Screenshots <!-- if applicable -->
```

Reply:
Added. Opted to reduce the headings since "additional details" likely can be covered under summary. Aside from that, can always add new headings as applicable.